librz/core/cfile.c

```diff
@@ -1600,7 +1603,13 @@ RZ_IPI void rz_core_io_file_open(RzCore *core, int fd) {
 	free(file);
 }
 
-RZ_IPI void rz_core_io_file_reopen(RzCore *core, int fd, int perms) {
+/**
+ * \brief Reopen file
+ * \param core RzCore instance
+ * \param fd File descriptor
+ * \param perms Permission \s RZ_PERM_R
+ */
+RZ_API void rz_core_io_file_reopen(RzCore *core, int fd, int perms) {
 	if (rz_io_reopen(core->io, fd, perms, 644)) {
```

Comment:
RZ_NONNULL and add a rz_return_if for core and fd

Reply:
`fd` is not a pointer.  So maybe it shouldn't be check in `rz_return_if`

Reply:
fd cannot be negative